src/Recipient.c

```diff
@@ -234,6 +234,7 @@ static bool HKDF_X(COSE * pCose, bool fHMAC, bool fECDH, bool fStatic, bool fSen
 
 bool _COSE_Recipient_decrypt(COSE_RecipientInfo * pRecip, COSE_RecipientInfo * pRecipUse, int algIn, size_t cbitKeyOut, byte * pbKeyOut, cose_errback * perr)
 {
+	UNUSED(pRecipUse);
```

Comment:
there is `UNUSED` and `UNUSED_PARAM`, can we remove one of those?

Reply:
Yes - UNUSED_PARAM can go away.